### ISSUE TITLE
visual: border is not complete "2;1H" is visible

### ISSUE THREAD

**Issue by budde25** [zellij-org/zellij#1107]
Thank you for taking the time to file this issue! Please follow the instructions and fill in the missing parts below the instructions, if it is meaningful. Try to be brief and concise.

**In Case of Graphical or Performance Issues**

logs, other one is empty
```
INFO   |zellij_client            | 2022-02-23 21:41:43.674 [main      ] [zellij-client/src/lib.rs:118]: Starting Zellij client!
INFO   |zellij_server            | 2022-02-23 21:41:43.679 [main      ] [zellij-server/src/lib.rs:186]: Starting Zellij server!
INFO   |zellij_server::wasm_vm   | 2022-02-23 21:41:43.736 [wasm      ] [zellij-server/src/wasm_vm.rs:86]: Wasm main thread starts
INFO   |zellij_server::wasm_vm   | 2022-02-23 21:41:45.070 [wasm      ] [zellij-server/src/wasm_vm.rs:226]: wasm main thread exits
INFO   |zellij_client            | 2022-02-23 21:41:45.072 [main      ] [zellij-client/src/lib.rs:342]: Bye from Zellij!
```

**Basic information**

`zellij --version`: zellij 0.25.0
`tput lines`: 26
`tput cols`: 136
`uname -av` or `ver`(Windows): 
Darwin Ethans-MacBook-Pro.local 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_ARM64_T6000 arm64

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:
None

**Further information**
It shows up on any shell I try zsh, bash, fish
the tput lines and cols did not matter
Also showed up in both Iterm2 and Terminal.app

<img width="1226" alt="image" src="https://user-images.githubusercontent.com/16231105/155454054-c18a1e05-aa8c-450c-950a-0e8720259174.png">

**Comment by mycraftmw** [comment]
Same here. I'm using Arch and launched zellij in yakuake.

**Comment by imsnif** [comment]
@budde25, @mycraftmw  - could you please follow the instructions in ISSUE_TEMPLATE.md and attach the debug log? This will allow me to reproduce this issue and fix it. Be sure to include the `tput lines` and `tput cols` values from within the Zellij pane that caused this.

Thanks!

**Comment by ozsay** [comment]
the fix for #816 solved this issue for me. if you want to confirm the fix as well, you can follow the comments on how to locally test that branch

**Comment by mycraftmw** [comment]
> @budde25, @mycraftmw - could you please follow the instructions in ISSUE_TEMPLATE.md and attach the debug log? This will allow me to reproduce this issue and fix it. Be sure to include the `tput lines` and `tput cols` values from within the Zellij pane that caused this.
> 
> Thanks!

/tmp/zellij-1000/zellij-log/zellij.log：
```
INFO   |zellij_client            | 2022-02-24 20:39:59.576 [main      ] [/home/tongyu/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/zellij-client-0.25.0/src/lib.rs:118]: Starting Zellij client! 
INFO   |zellij_server            | 2022-02-24 20:39:59.579 [main      ] [/home/tongyu/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/zellij-server-0.25.0/src/lib.rs:186]: Starting Zellij server! 
INFO   |zellij_server::wasm_vm   | 2022-02-24 20:39:59.631 [wasm      ] [/home/tongyu/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/zellij-server-0.25.0/src/wasm_vm.rs:86]: Wasm main thread starts 
INFO   |zellij_server::wasm_vm   | 2022-02-24 20:40:43.740 [wasm      ] [/home/tongyu/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/zellij-server-0.25.0/src/wasm_vm.rs:226]: wasm main thread exits 
INFO   |zellij_client            | 2022-02-24 20:40:43.740 [main      ] [/home/tongyu/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/zellij-client-0.25.0/src/lib.rs:342]: Bye from Zellij! 
```

/tmp/zellij-1000/zellij-log/zellij-10.log
```
]0;tongyu@Tongyu-arch:~[?2004h[tongyu@Tongyu-arch ~]$ 
[K[tongyu@Tongyu-arch ~]$ 
```

`zellij --version`: zellij 0.25.0
`tput lines`: 54
`tput cols`: 209
`uname -av` : Linux Tongyu-arch 5.16.10-arch1-1 #1 SMP PREEMPT Wed, 16 Feb 2022 19:35:18 +0000 x86_64 GNU/Linux

![ze](https://user-images.githubusercontent.com/8987163/155530119-0928bc07-99ce-48f9-b69f-4d739f36adaa.png)

Just start zellij, this will show up. As you can see in the screenshot, the first two lines are broken.
